apps/data/config/test.exs

```diff
@@ -8,3 +8,5 @@ config :data, Data.Repo,
   database: "extracurricular_test",
   hostname: "localhost",
   pool: Ecto.Adapters.SQL.Sandbox
+
+config :logger, level: :warn
```

Comment:
Test output was getting really noisy with some debug-level sql logging so I added this.